generator/src/main/java/com/scottlogic/deg/generator/GenerateExecute.java

```diff
@@ -43,7 +46,7 @@ public GenerateExecute(GenerationConfig config,
     public void run() {
 
         ValidationResult validationResult = validator.preProfileChecks(config, configSource);
-
+        validationResult.addErrorMessages(profileValidator.validateProfile(configSource.getProfileFile()).errorMessages);
```

Comment:
This is a whacking long line... not sure its the best way to go.

How about this approach

```suggestion
        ValidationResult profileValidationResult = profileValidator.validateProfile(configSource.getProfileFile());
        
        if (!validationResult.isValid() || !profileValidationResult.isValid()) {
          errorReporter.display(validationResult);
          errorReporter.display(profileValidationResult);
```

Reply:
changed this, but it means that making the schema profile optional will result in two `if(!result.isValid()` checks